packages/components/app/styles/components/stepper/indicator/task.scss

```
@@ -0,0 +1,63 @@
+//
+// STEPPER > INDICATOR > TASK
+//
+// properties within each class are sorted alphabetically
+//
+
+$hds-stepper-indicator-task-statuses: ( "incomplete", "progress", "processing", "complete" );
+$hds-stepper-indicator-task-size: 16px;
+
+// Determine states and corresponding styles
+$status-props: (
+    "incomplete": (
+        "color-default": var(--token-color-palette-neutral-300),
+        "color-hover": var(--token-color-palette-blue-200),
+        "color-active": var(--token-color-palette-blue-300),
+    ),
+    "progress": (
+        "color-default": var(--token-color-palette-blue-200),
+        "color-hover": var(--token-color-palette-blue-300),
+        "color-active": var(--token-color-palette-blue-400),
+    ),
+    "processing": (
+        "color-default": var(--token-color-palette-blue-200),
+    ),
+    "complete": (
+        "color-default": var(--token-color-palette-green-200),
+        "color-hover": var(--token-color-palette-green-300),
+        "color-active": var(--token-color-palette-green-400),
+    )
+);
+
+// Base styling for indicator::task
+.hds-stepper-indicator-task {
+    width: $hds-stepper-indicator-task-size;
+    height: $hds-stepper-indicator-task-size;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: var(--token-color-foreground-strong);
+}
+
+.hds-stepper-indicator-task__icon {
+    color: inherit;
+    width: 12px;
+    height: 12px;
+}
+
+.hds-stepper-indicator-task--is-interactive {
+    cursor: pointer;
+
+    @each $status in $hds-stepper-indicator-task-statuses {
+        // For each status set the icon color based on the $status-props
+        &.hds-stepper-indicator-task--status-#{$status} {
+            color: map-get($status-props, $status, "color-default");
+            &:hover, &.mock-hover { color: map-get($status-props, $status, "color-hover")};
+            &:active, &.mock-active { color: map-get($status-props, $status, "color-active")};
```

better to move on multiple lines, it's more readable (it would have been easier to spot that a couple of semicolons are in the wrong place)

```suggestion
            color: map-get($status-props, $status, "color-default");

            &:hover,
            &.mock-hover {
                color: map-get($status-props, $status, "color-hover");
            }

            &:active,
            &.mock-active {
                color: map-get($status-props, $status, "color-active");
            }
```